src/wiki/core/plugins/registry.py

```diff
@@ -48,6 +50,19 @@ def register(PluginClass):
             'markdown_extensions',
             []))
 
+    _html_whitelist.extend(
+        getattr(
+            PluginClass,
+            'html_whitelist',
+            []))
+
+    # TODO: should we sanitize the list somehow or do we trust plugins to 100%?
```

Comment:
It's not so much about trusting plugins - they are executable code sitting on the server!! :) It's about trusting that user input can't be made malicious.

If a plugin decides to allow everything, well.. so could the user in the settings. So it's the same idea - with great reconfigurability comes great responsibility :)

Reply:
right, I removed the comment